packages/service-proxy/src/__tests__/unit/mixin/service.mixin.unit.ts

```diff
@@ -3,7 +3,7 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {Application, Component, Provider, BindingScope} from '@loopback/core';
+import {Application, BindingScope, Component, Provider} from '@loopback/core';
```

Comment:
Since there are no other changes in this file, I think it would be better to revert this reorder of imports.